en/SUMMARY.md

```diff
@@ -339,18 +292,67 @@
+      * [VESC](peripherals/vesc.md)
+  * [Data Links](data_links/README.md)
+    * [FrSky Telemetry](peripherals/frsky_telemetry.md)
+    * [Satellite Coms (Iridium/RockBlock)](advanced_features/satcom_roadblock.md)
+    * [MAVLink Telemetry (OSD/GCS)](peripherals/mavlink_peripherals.md)
+    * [Telemetry Radios](telemetry/README.md)
+      * [SiK Radio](telemetry/sik_radio.md)
+        * [RFD900 (SiK) Telemetry Radio](telemetry/rfd900_telemetry.md)
+        * [HolyBro (SIK) Telemetry Radio](telemetry/holybro_sik_radio.md)
+      * [Telemetry Wifi](telemetry/telemetry_wifi.md)
+        * [ESP8266 WiFi Module](telemetry/esp8266_wifi_module.md)
+        * [ESP32 WiFi Module](telemetry/esp32_wifi_module.md)
+        * [3DR Telemetry Wifi (Discontinued)](telemetry/3dr_telemetry_wifi.md)
+      * [Microhard Serial Telemetry Radio](telemetry/microhard_serial.md)
+        * [ARK Electron Microhard Serial Telemetry Radio](telemetry/ark_microhard_serial.md)
+        * [Holybro Microhard P900 Telemetry Radio](telemetry/holybro_microhard_p900_radio.md)
+      * [CUAV P8 Telemetry Radio](telemetry/cuav_p8_radio.md)
+      * [HolyBro XBP9X - Discontinued](telemetry/holybro_xbp9x_radio.md)
+    * [TBS Crossfire (CRSF) Telemetry](telemetry/crsf_telemetry.md)
+  * [Power Modules/PDB](power_module/README.md)
+    * [CUAV HV pm](power_module/cuav_hv_pm.md)
+    * [CUAV CAN PMU](dronecan/cuav_can_pmu.md)
+    * [Holybro PM02](power_module/holybro_pm02.md)
+    * [Holybro PM07](power_module/holybro_pm07_pixhawk4_power_module.md)
+    * [Holybro PM06 V2](power_module/holybro_pm06_pixhawk4mini_power_module.md)
+    * [Holybro PM02D (digital)](power_module/holybro_pm02d.md)
+    * [Holybro PM03D (digital)](power_module/holybro_pm03d.md)
+    * [Pomegranate Systems Power Module](dronecan/pomegranate_systems_pm.md)
+    * [Sky-Drones SmartAP PDB](power_module/sky-drones_smartap-pdb.md)
+  * [Smart Batteries](smart_batteries/README.md)
+    * [Rotoye Batmon Battery Smartification Kit](smart_batteries/rotoye_batmon.md)
+  * [Peripherals](peripherals/README.md)
+    * [ADSB/FLARM (Traffic Avoidance)](peripherals/adsb_flarm.md)
+    * [Air Traffic Avoidance: ADSB/FLARM](advanced_features/traffic_avoidance_adsb.md)
+    * [Air Traffic Avoidance: UTM](advanced_features/traffic_avoidance_utm.md)
+    * [Camera](peripherals/camera.md)
+    * [Gimbal \(Mount\) Configuration](advanced/gimbal_control.md)
+    * [Grippers](peripherals/gripper.md)
+      * [Servo Gripper](peripherals/gripper_servo.md)
+    * [Parachute](peripherals/parachute.md)
     * [I2C Peripherals](sensor_bus/i2c_general.md)
       * [I2C bus accelerators](sensor_bus/i2c_general.md#i2c-bus-accelerators)
       * [TFI2CADT01 I2C address translator](sensor_bus/translator_tfi2cadt.md)
     * [CAN Peripherals](can/README.md)
     * [DroneCAN Peripherals](dronecan/README.md)
       * [PX4 DroneCAN Firmware](dronecan/px4_cannode_fw.md)
       * [ARK CANnode](dronecan/ark_cannode.md)
```

Comment:
I wonder if we should move Ethernet Setup down here with the other "BUS" stuff. Inclined not to.